browser/resources/settings/brave_overrides/site_details.js

```diff
@@ -43,5 +43,23 @@ RegisterPolymerTemplateModifications({
         `)
       }
     }
+    const usageSection = templateContent.querySelector('div#usage')    
```

Comment:
nit: please add a log if the element was not found. This avoids searching for the problem on chromium rebase when original line numbers are missing.

Reply:
done